lab_1/main.py

```diff
@@ -87,25 +146,36 @@ def get_adjacent_words(tokens: list, word: str, left_n: int, right_n: int) -> li
     right_n = 3
     --> [['man', 'is'], ['dog, 'cat']]
     """
-    pass
+    contexts = get_concordance(tokens, word, left_n, right_n)
+    adjacent_words = []
+    for context in contexts:
+        if left_n == 0:
```

Comment:
No need for explicit check.
You can just `if not var: ...`

Reply:
Fixed. Thank you!